.github/workflows/report.yml

```diff
@@ -0,0 +1,17 @@
+name: Report
+on:
+  workflow_run:
+    workflows: [ 'Continuous Integration' ]
+    types:
+      - completed
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion != 'cancelled' }}
+    steps:
+      - name: Report
```

Comment:
Ditto. This will fail as this spawns a entirely new environment. Download the artifacts uploaded by build then report from there.